docs/api-reference/io-columns-ranking.md

```diff
@@ -1,6 +1,5 @@
 ### Input and Output Columns
-The input label column data must be <xref:System.Single> and input group column
-data must be <xref:System.UInt32>. This trainer outputs the following columns:
+The input label column data must be <xref:System.Single> or [key](xref:Microsoft.ML.Data.KeyDataViewType) type, the feature column must be a known-sized vector of <xref:System.Single> and input row group column must be [key](xref:Microsoft.ML.Data.KeyDataViewType) type. This trainer outputs the following columns:
 
```

Comment:
Please replace input description with this:

The input label data type must be [key](xref:Microsoft.ML.Data.KeyDataViewType) type or <xref:System.Single>. The value of the label determines relevance, where higher values indicate higher relevance. If the label is a key, then the key index is the relevance value, where the 0-index is the least relevant. If the label is a <xref:System.Single>, larger values indicate higher relevance.